pcluster/pcluster_stack.py

```diff
@@ -200,19 +286,23 @@ def __init__(self, scope: cdk.Construct, id: str, **kwargs) -> None:
                                )
                            )
 
-        create_user = cdk.CfnParameter(self, "CreateUser", default="false", type="String", allowed_values=['true','false']).value_as_string
+        #create_user = cdk.CfnParameter(self, "CreateUser", default="true", type="String", allowed_values=['true','false']).value_as_string
+        create_user = "true"
```

Comment:
Did we go back on the create user option?

Reply:
Temporarily. In the next push, I've got code that will enable it.